apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.cs

```diff
@@ -163,6 +158,7 @@ internal static UrlSigner FromCredential(IHttpExecuteInterceptor credential) =>
         /// Creates a new <see cref="UrlSigner"/> instance from the JSON configuration file of a Google credential.
         /// See <see cref="FromCredential(GoogleCredential)"/> for more information about supported credential types.
         /// </summary>
+        [Obsolete("This method is being deprecated because of a potential security risk. Instead use one of the FromCredential methods.")]
```

Comment:
Can you add remarks to this method linking to the documentation page similar to what we have on Auth and GAX? Just copy paste it from there.